shell/src/main/java/alluxio/shell/AlluxioShell.java

```diff
@@ -43,15 +43,13 @@
 @NotThreadSafe
 public class AlluxioShell implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
-  private static final HashMap<String, String[]> CMD_ALIAS = new HashMap<String, String[]>() {
-    {
-      put("chgrpr", new String[] {"chgrp", "-R"});
-      put("chmodr", new String[] {"chmod", "-R"});
-      put("chownr", new String[] {"chown", "-R"});
-      put("lsr", new String[] {"ls", "-R"});
-      put("rmr", new String[] {"rm", "-R"});
-    }
-  };
+  private static final Map<String, String[]> CMD_ALIAS = ImmutableMap.<String, String[]> builder()
```

Comment:
checkstyle complains about: error: GenericWhitespace '>' is followed by whitespace.